core/src/main/scala/cats/data/Streaming.scala

```diff
@@ -812,7 +812,7 @@ object Streaming extends StreamingInstances {
    * traversals produce the same results.
```

Comment:
I think this doc needs to be updated to no longer say that the result is memoized.